File: Java/src/main/java/com/contidio/sdk/exceptions/StopExecutionBackendException.java
package com.contidio.sdk.exceptions;

import org.apache.commons.httpclient.HttpStatus;

import com.contidio.sdk.proto.MetadataProtocol.BackendErrorCode;

@SuppressWarnings("serial")
public class StopExecutionBackendException extends BackendException {

	public StopExecutionBackendException() {
		super(BackendErrorCode.STOP_EXECUTION, "Abandon this task", "StopExecutionBackendException", 
				HttpStatus.SC_NOT_ACCEPTABLE);
	}

}

File: .NET Core/Exceptions/StopExecutionBackendException.cs
using com.contidio.sdk.proto;
using System.Net;

namespace Contidio.Sdk.Exceptions
{
    public class StopExecutionBackendException : BackendException
    {
        public StopExecutionBackendException()
            : base(BackendErrorCode.STOP_EXECUTION, "Abandon this task", "StopExecutionBackendException", HttpStatusCode.NotAcceptable)
        {
        }
    }
}
